packages/office-ui-fabric-react/src/components/Label/examples/Label.Basic.Example.tsx

```diff
@@ -1,3 +1,4 @@
+// @codepen
```

Comment:
Wouldn't this show up on the website too and be potentially confusing?

Reply:
I guess this is probably the most direct mechanism though..

Reply:
yes, this will unfortunately show up on the website. We do this with disabling tslint in some examples as well, and those appear on the website too. It does seem like the most direct way to go about tagging examples though..